torch/_dynamo/utils.py

```diff
@@ -921,6 +962,8 @@ def us_to_ms(field):
         metric = metrics.get(field, None)
         return metric // 1000 if metric is not None else None
 
+    # Log inductor_config to compilation metrics
+    metrics["inductor_config"] = _scrubbed_inductor_config_for_logging()
```

Comment:
Instead of mutating the param, can you instead just add this to the `legacy_metrics` dict? We can rename it to `common_metrics` maybe. Something like:

```
common_metrics = {
    "inductor_config": _scrubbed_inductor_config_for_logging(),
    ... any future common metrics would go here ...
    # TODO: Temporary; populate legacy fields from their replacements.
    # Remove when we decide we can really deprecate them.
    ... all the existing "legacy" fields go here ...
 }
```